lib/compat/wordpress-6.1/blocks.php

```diff
@@ -158,3 +158,49 @@ function gutenberg_block_type_metadata_multiple_view_scripts( $metadata ) {
 	return $metadata;
 }
 add_filter( 'block_type_metadata', 'gutenberg_block_type_metadata_multiple_view_scripts' );
+
+
+/**
+ * Allows multiple block styles.
+ *
+ * This is a duplicate of _wp_multiple_block_styles, except that it removes the line that sets the key to the first item in the array.
+ * We also check that $handle is a string.
```

Comment:
What line is this? Can you include an explicit reference? Ideally I won't have to parse the code and learn context to appreciate what this does.

Reply:
Here's the ref

https://github.com/WordPress/wordpress-develop/blob/6dab53e93bb948a59e48b5856208946d9ee83e74/src/wp-includes/blocks.php#L1267